core/src/codegen/from_meta_impl.rs

```diff
@@ -113,6 +122,7 @@ impl<'a> ToTokens for FromMetaImpl<'a> {
                     fn from_string(lit: &str) -> ::darling::Result<Self> {
                         match lit {
                             #(#unit_arms)*
+                            #unit_arm_default
```

Comment:
This change feels orthogonal to the heterogeneous arms improvement; can we split them into two PRs?

Reply:
makes sense ... will do.